### PR TITLE
Added a shell script to run bot daily

### PR DESCRIPTION
**Pull request by SubhrajitPrusty**
updated README, with instructions to do so

### CODE REVIEW

File: schedule.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+COUNT=0
+while [ true ]; do
```
Comment (icyflame) [inline schedule.sh]: Don't use while in a script.

Instead you can set the Cron timeout appropriately to achieve the same effect. You don't really need a script. You only need to add the cron job line for this:

```sh
86400 * * * * cd /path/ && python chitragupta.py
```

cc @zorroblue

Comment (icyflame) [inline schedule.sh:4]: Don't use while in a script.

Instead you can set the Cron timeout appropriately to achieve the same effect. You don't really need a script. You only need to add the cron job line for this:

```sh
86400 * * * * cd /path/ && python chitragupta.py
```

cc @zorroblue

**Review by icyflame — CHANGES_REQUESTED** [review]

**Comment by SubhrajitPrusty** [comment]
In #4 it is said to use a script, that's why I used that. Like I mentioned there, I didn't use crontab or atd, assuming insufficient permissions for the user, or `atd` not running.

**Comment by zorroblue** [comment]
Can you look into why there are 5 files shown as changed? I assume this would change only the README and the additional script you added?

**Comment by SubhrajitPrusty** [comment]
I really don't know. Also one of my commits shows as unknown user, so it might be a glitch/bug with git.

Furthermore, I'll close this PR and send a clean one.

**Pull request closed without merging**